api/transactions/types.go

```diff
@@ -47,19 +47,21 @@ func (c *Clause) String() string {
 
 // Transaction transaction
 type Transaction struct {
-	ID           thor.Bytes32        `json:"id"`
-	ChainTag     byte                `json:"chainTag"`
-	BlockRef     string              `json:"blockRef"`
-	Expiration   uint32              `json:"expiration"`
-	Clauses      Clauses             `json:"clauses"`
-	GasPriceCoef uint8               `json:"gasPriceCoef"`
-	Gas          uint64              `json:"gas"`
-	Origin       thor.Address        `json:"origin"`
-	Delegator    *thor.Address       `json:"delegator"`
-	Nonce        math.HexOrDecimal64 `json:"nonce"`
-	DependsOn    *thor.Bytes32       `json:"dependsOn"`
-	Size         uint32              `json:"size"`
-	Meta         *TxMeta             `json:"meta"`
+	ID                   thor.Bytes32        `json:"id"`
+	ChainTag             byte                `json:"chainTag"`
+	BlockRef             string              `json:"blockRef"`
+	Expiration           uint32              `json:"expiration"`
+	Clauses              Clauses             `json:"clauses"`
+	GasPriceCoef         uint8               `json:"gasPriceCoef"`
```

Comment:
How does geth handles this ? 
It returns a transaction object with all fields ?

Reply:
Yes, in their case the `gasPriceCoef` would be `gasPrice` and our `maxFeePerGas` and `maxPriorityFeePerGas` are respectively `gasFeeCap` and `gasTipCap` and they are all in the same [entity](https://github.com/ethereum/go-ethereum/blob/master/internal/ethapi/api.go#L1104) that is returned by the rpc method